main.go

```diff
@@ -58,8 +59,9 @@ func (c *component) New(ctx context.Context, conf *config) (func(context.Context
 		LogFn:           domain.LoggerFromContext,
 		StatFn:          domain.StatFromContext,
 		AssetAttributor: a,
-		// AttributionFailureHandler field is simply a stub, it varies org to org
+		// AttributionFailureHandler and AttributedAssetValidator fields are stubs, it varies company to company
```

Comment:
Suggestion: I think adding information about how one would go about adding AttributionFailureHandlers and AttributedAssetValidators in the README would be useful.

Reply:
i can write some stuff on the readme